src/main/java/frc/robot/RobotConstants.java

```diff
@@ -56,9 +56,12 @@ public interface APRILTAGIDS {
     		public static final int CURRENT_LIMIT_AMPS = 8;
     		public static final int MAX_SPIKE_CURRENT = 12;
     		public static final double COUNTS_PER_METER = 0.004826; 
+    		
     	} 
     	public interface HOMING{
     		public static final double HOMING_SPEED_PERCENT = 0.35;
+    		public static final double HOMING_SPEED_VELOCITY = 5.0; //i think this is degrees/sec
```

Comment:
I think this might be in rev/min.  See the Description on the page:  https://github.com/REVrobotics/SPARK-MAX-Examples/tree/master/Java/Velocity%20Closed%20Loop%20Control

Reply:
@mandrews281 oh that helps!  

we are setting the velocityconversion factor like this:

`	    telescopeMotor.getEncoder().setVelocityConversionFactor(ARM.SETTINGS.COUNTS_PER_METER);`

using the same conversion factor as for position.  if the natural units are rpm, i think our conversion factor is 'elbown degrees per motor rev'. so that means we'll probably get 5 'elbow degrees per minute' , which is definitely way too slow.